Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/SplitBase.cs

```diff
@@ -0,0 +1,220 @@
+        /// Gets or sets the content template for the <see cref="Content"/>.
+        /// </summary>
+        public DataTemplate ContentTemplate
+        {
+            get { return (DataTemplate)GetValue(ContentTemplateProperty); }
+            set { SetValue(ContentTemplateProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="ContentTemplate"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ContentTemplateProperty =
+            DependencyProperty.Register(nameof(ContentTemplate), typeof(DataTemplate), typeof(SplitBase), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the cursor to use when hovering over the sizer.
+        /// </summary>
+        public CoreCursorType GripperCursor
+        {
+            get { return (CoreCursorType)GetValue(GripperCursorProperty); }
+            set { SetValue(GripperCursorProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="GripperCursor"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty GripperCursorProperty =
+            DependencyProperty.Register(nameof(GripperCursor), typeof(CoreCursorType), typeof(SplitBase), new PropertyMetadata(CoreCursorType.SizeWestEast));
+
+        /// <summary>
+        /// Gets or sets the foreground color of sizer grip.
+        /// </summary>
+        public Brush GripperForeground
+        {
+            get { return (Brush)GetValue(GripperForegroundProperty); }
+            set { SetValue(GripperForegroundProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="GripperForeground"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty GripperForegroundProperty =
+            DependencyProperty.Register(nameof(GripperForeground), typeof(Brush), typeof(SplitBase), new PropertyMetadata(default(Brush)));
+
+        /// <summary>
+        /// Gets or sets the direction that the sizer will interact with.
+        /// </summary>
+        public ContentResizeDirection ResizeDirection
+        {
+            get { return (ContentResizeDirection)GetValue(ResizeDirectionProperty); }
+            set { SetValue(ResizeDirectionProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="ResizeDirection"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty ResizeDirectionProperty =
+            DependencyProperty.Register(nameof(ResizeDirection), typeof(ContentResizeDirection), typeof(SplitBase), new PropertyMetadata(ContentResizeDirection.Vertical));
+
+        /// <summary>
+        /// Gets or sets the control that the <see cref="SplitBase"/> is resizing. Be default, this will be the visual ancestor of the <see cref="SplitBase"/>.
+        /// </summary>
+        public FrameworkElement TargetControl
```

Comment:
I know there was a similar property in GridSplitter but it was private for internal use, feel like we should just make it a private field in GridSplitter and have this property in ContentSizer vs. using `new` to try and override/hide it?